modules/mobile/app/workers/mobile/v0/vaccines_updater_job.rb

```diff
@@ -18,7 +17,7 @@ def perform
         aggregate = {}
         results = { created: 0, updated: 0, persisted: 0 }
 
-        group_name_xml.root.children.each do |node|
+        manufacturer_xml.root.children.each do |node|
```

Comment:
Does this result in any changes in the data? It would be good to run this script with both sources and confirm that they produce identical outputs.

Reply:
Good point. There were some vaccine records before that only had a group name and cvx code but no manufacturer so those will no longer be tracked. Comparing outputs will be a good idea to ensure the group names are the same coming from LH compared to this group name xml as well as the manufacturers but the covid manufacturers that are used theoretically should be unchanged. I'll check it out.

Reply:
It sounds like we may end up with fewer vaccines using this approach. However, we aren't destroying old vaccine records, so they'll still be around in production. It might be worth some sentry logging in the adapter if the vaccine isn't found, since that may be a sign that we're not getting all the data we need from the manufacturers' table. We probably should've done it before, just to have an indicator of how complete our data is.

Reply:
If we did get a miss on the covid vaccine lookup, would it be worth interrupting flow to raise a sentry error? the result of a miss would be no manufacturer shown which is preferable to no information about the immunization at all. May be better to add a datadog metric to the vaccine dashboard and log the issue instead. Thoughts?

Reply:
You can log to sentry without raising an error. There's a SentryLogging module that might give you some ideas. I haven't looked closely at it, but it looks like we do it via the rails logger. In the past, I've used Sentry directly to log errors without disrupting the code processes, but I don't see any evidence of us doing that so I would just follow the pattern that's been established. 

That's assuming we want to do this at all. It might be useful to know when we don't find a match, but obviously the code works fine without it.

Reply:
That's good to know about sentry logging but I agree, if no ones using sentry for that, it may add some confusion on whats showing up on sentry. I ended up adding a metric to give a count of covid manufactures missing just so we can get an indicator to check out the latest manufacturer xml.